draft-ietf-scone-protocol.md

```diff
@@ -528,6 +528,32 @@ a processing error at network elements. For example, they might pick connection
 identifiers of arbitrary length. Network elements can mitigate these attacks
 with implementations that fully conform to the specification of {{packet}}.
 
+## Damage to Other Protocols
+
+Network elements that update SCONE packet fields might do that for datagrams
+exchanged in other protocols.
+This could result in damage to those protocols.
+
+The most serious damage occurs when every datagram is modified,
+because that could mean that the protocol is
+effectively unable to operate end-to-end.
+
+To that end, network elements MUST only update the content of datagrams
+on a given address tuple
+more than a few times each enforcement period.
+Network elements MAY update more often
+immediately after a change in their throughput advice,
+to reduce the reaction time from senders.
+
```

Comment:
OK, we are down to editorial. I suggested "SHOULD NOT ... more than a few times". You propose "MUST only ... more than a few times." I still prefer my original proposal for two reasons:

- "MUST only ... more" is a weird construct. I think this should be either "MUST NOT ... more" or "SHOULD NOT .. more".
- I originally thought of writing "MUST NOT", but in general if we have a combination of mandate and exception we write "SHOULD NOT \<do something>" followed by "MAY \<by exception do something else>", so I would rather fit the established pattern.

Reply:
Done.  Thanks.